ROADMAP.md

```diff
@@ -201,7 +201,7 @@ ___
 **NEXT**
 * [ ] Add [Coinpaprika](https://coinpaprika.com/api/)
 * [ ] Add crypto charts
-* [ ] Add binance 
+* [x] Add binance (@jmaslek)
```

Comment:
You should move this to the part above (to the "done" area) and add your PR number